components/script/dom/cssstyledeclaration.rs

```diff
@@ -340,6 +342,46 @@ impl CSSStyleDeclarationMethods for CSSStyleDeclaration {
         rval
     }
 
-    // https://drafts.csswg.org/cssom/#cssstyledeclaration
+    // https://drafts.csswg.org/cssom/#dom-cssstyledeclaration-csstext
+    fn CssText(&self) -> DOMString {
+        let elem = self.owner.upcast::<Element>();
+        let style_attribute = elem.style_attribute().borrow();
+
+        if let Some(declarations) = style_attribute.as_ref() {
+            DOMString::from(declarations.to_css_string())
+        } else {
+            DOMString::new()
+        }
+    }
+
+    // https://drafts.csswg.org/cssom/#dom-cssstyledeclaration-csstext
+    fn SetCssText(&self, value: DOMString) -> ErrorResult {
```

Comment:
This implementation is kind of ugly since it can update the inline style twice (triggering an extra paint?) when cssText parses into declarations.

Also, the parsing returns `Arc`s that we immediately unwrap and doesn't preserve specified order as far as I can tell.